util.go

```diff
@@ -37,3 +37,37 @@ func ReadPublicKeyFromEnv(keyid, issuer string) (ed25519.PublicKey, error) {
 
 	return ed25519.PublicKey(keyBytes), nil
 }
+
+// CreateIdentityTokenAddressedTo returns a Replit identity token that proves this Repl's identity
+// that includes an audience claim to restrict forwarding.
```

Comment:
Maybe add a little blurb about this being a bit slower than creating a `NewSigningAuthority` if one wants to create multiple tokens.

Reply:
did so, added a function to easily create the authority, too.